### DISCUSSION TITLE
dest performance ...

### DISCUSSION THREAD

**Discussion by MortThePersian** [XTLS/Xray-core#1935] [Q&A]
Dear Sisters & Brothers;

Is there a effective way to choose a website for  Reality ?
We, in Iran tested some & it resulted in interesting performance differences that raised this important question !
What are your recommendations in this regard ?!

P.S. clearance by RPRX would be greatly appreciated, We are in a unexplainable confusion !

Sincerely

**Comment by picklefan** [comment]
https://github.com/XTLS/RealiTLScanner

 The site needs to be using TLS 1.3 for connection and x25519 as key exchange mechanism and can not be redirecting sites

**Reply by MortThePersian** [reply]
Thank you ! didn't know these rules ...

**Reply by GreatMichaelLee** [reply]
what is this script used for? Scanning a C subnet IP range that to find out which one is running TLS1.3 and h2? If yes, it's SNI is definitely not allowed in dest server( the -addr specified domain),but I remember that the IP is more similar the safety is more better...how it will work for reality? confused....
